libraries/hatch-i18n/src/i18n-utils.ts

```diff
@@ -30,6 +30,9 @@ export const initializeTranslations = async (translations: Resource) => {
             if (Interval.isInterval(value)) {
               return String(Math.floor(value.length(format as DurationUnit)));
             }
+            if (Array.isArray(value) && value.every(DateTime.isDateTime)) {
+              return value.map((element: DateTime) => element.toFormat(format)).join(', ');
```

Comment:
I don't think it's safe to assume this is how it should be joined for all languages

Reply:
updated

Reply:
updated